examples/rosettacode/rock_paper_scissor.nit

```diff
@@ -0,0 +1,50 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+# This program is public domain
+#
+# Task: Programming_Task/Rock-paper-scissors
+# SEE: <http://rosettacode.org/wiki/Rock-paper-scissors>
+#
+# author: Istvan szalai 22/05/2015
+
+#return a value to dertermine who win the match
```

Comment:
Furthermore, this comment should explain your logic and why you return 0, 1 or 2.